modules/volunteers/admin/vue/hourEntryField.js

```diff
@@ -38,28 +28,22 @@ export default {
     },
     hours: {
       get() {
-        if (this.enterHours != null) {
-          const v = this.enterHours;
-          this.enterHours = null;
-          return v;
-        }
         return Math.floor(this.modelValue);
       },
       set(value) {
-        this.enterHours = Math.floor(value);
+        var enterHours = Math.floor(value);
+        value = enterHours + (this.modelValue % 1)
+        this.$emit('update:modelValue', value);
```

Comment:
It looks like changing `value` should automatically result in an emit call, right? Or is it not triggering `computed.value.set()`?

Reply:
Maybe... It has been forever since I authored this and I do not recall seeing it happen but I may have missed it.